benchmark/python/prompts.json

```diff
@@ -0,0 +1,7 @@
+{
+    "16": "How are astronauts launched into space quickly on those rockets? ",
+    "64": "",

```

Comment:
Should there be prompts for the larger sizes?

Reply:
I left that part not committed, I can add sample prompts as well.

Reply:
added